local-notifications/README.md

```diff
@@ -10,6 +10,7 @@ npx cap sync
 ```
 
 ## Android
+Android 13 requires a permission check in order to send notifications.  You are required to call `checkPermissions()` and `requestPermissions()` accordingly, when targeting SDK 33.
```

Comment:
```suggestion

Android 13 requires a permission check in order to send notifications.  You are required to call `checkPermissions()` and `requestPermissions()` accordingly.

On Android 12 and older it won't show a prompt and just return as granted.
```

Added some spacing between the title and the text
Also removed the note about targeting SDK 33 as that's going to be mandatory.
And added a note about Android 12 and older not prompting since it's a common question why requesting the permission doesn't prompt.